ai_edge_torch/generative/layers/unet/model_config.py

```diff
@@ -269,3 +278,11 @@ class DiffusionModelConfig:
 
   # Activation type used in final layer
   final_activation_type: layers_cfg.ActivationType
+
+  # Additional parameters
+  attention_hidden_channels: Optional[List[int]] = None
```

Comment:
Could you add comments for each of the field?

Reply:
+1

Reply:
Removed it, actually not used in blocks